heuristic/local_search_operators.py

```diff
@@ -9,7 +9,7 @@
     calculate_isolated_working_days,
     calculate_partial_weekends,
     calculate_weekly_rest,
-    delta_calculate_deviation_from_contracted_hours,
```

Comment:
Is this replaced with a new function that calculates the deviation from contracted hours?

Reply:
Good catch! I forgot to add an explanation for this. The thing is that `delta_calculate_deviation_from_contracted_hours` does not exist in `master`. I was unsure if it was because you had deleted the function but forgot the function calls, or if this was because of merge problems. 

There exists a similar function. Is this the same thing so I should undelete the function calls and rename them?
<img width="1118" alt="bilde" src="https://user-images.githubusercontent.com/1371159/81476766-e6a79e00-9213-11ea-8b51-eaf1a8f2c91b.png">

Reply:
Yes. This might be my bad. I have forgotten to remove unused imports. The correct function seem to be delta_calculate_negative_deviation_from_contracted_hours. This should be imported if it isn't already